.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
 .theos
-packages/
+packages
```

Comment:
I want keep the slash. It indicates that it's a folder.

Reply:
Yes. You're right.